tests/integration/test_vulnerability_detector/test_feeds/test_download_feeds.py

```diff
@@ -60,14 +59,14 @@
     for os in values['os']:
         parameters.append({'PROVIDER_NAME': provider, 'OS': os, 'UPDATE_FROM_YEAR': values['update_from_year']})
         metadata.append({'provider_name': provider, 'os': os, 'update_from_year': values['update_from_year'],
-                     'system_log': values['system_log'], 'download_timeout': values['download_timeout']})
+                         'system_log': values['system_log'], 'download_timeout': values['download_timeout']})
         ids.append(f"{provider}_{os}")
 
-  # If empty os list
+    # If empty os list
     if len(values['os']) == 0:
         parameters.append({'PROVIDER_NAME': provider, 'OS': '', 'UPDATE_FROM_YEAR': values['update_from_year']})
         metadata.append({'provider_name': provider, 'os': '', 'update_from_year': values['update_from_year'],
-                     'system_log': values['system_log'], 'download_timeout': values['download_timeout']})
+                         'system_log': values['system_log'], 'download_timeout': values['download_timeout']})
         ids.append(f"{provider}")
```

Comment:
This isn't really necessary. The provider is already a string so you can directly append it to the list.

Reply:
Done ca26474b64f76ba719b585d27df0f1728233f213